Audio/AudioEngine.cpp

```diff
@@ -277,24 +277,10 @@ class AudioEngine::Impl
         mEngineFlags(AudioEngine_Default),
         mCategory(AudioCategory_GameEffects),
         mVoiceInstances(0)
-    #ifdef USING_XAUDIO2_7_DIRECTX
-        , mDLL(nullptr)
```

Comment:
Another good reason to drop XAudio2.7: this DLL loader bug required a nasty hack.